api/src/lib.rs

```diff
@@ -2540,6 +2540,143 @@ ffi_export!(
     }
 );
 
+ffi_export!(
+    fn Java_com_horizen_certnative_NaiveThresholdSignatureWKeyRotation_nativeCreateMsgToSign(
```

Comment:
It seems to me the same exact implementation as in `Java_com_horizen_certnative_NaiveThresholdSignature_nativeCreateMsgToSign` so let's save this additional code by calling NaiveThresholdSig.createMsgToSign from NaiveThresholdSigWKeyRotation.createMsgToSign

Reply:
Done